frontend/src/components/account-mgmt/ManageUserDialog.vue

```diff
@@ -382,6 +402,56 @@ export default {
         this.selectedFacilityIds = this.facilitiesToAdminister?.map((facility) => facility.facilityId)
       }
     },
+
+    /**
+     * Check if BCeID exists in provider organization, if so raise error message.
+     */
+    async checkBCeIDExists(userName) {
+      try {
+        if (this.user.userName) {
+          const res = await ApiService.apiAxios.get(`${ApiRoutes.USER}/${userName}?providerProfile=false`);
+          if (res.data.length >= 1) {
+            this.errorMessages = ['A user with this BCeID already exists.'];
+          } else {
+            this.errorMessages = [];
+          }
+        }
+      } catch (error) {
+        if (error.response.status === 404) {
+          this.errorMessages = [];
+        } else {
+          this.setFailureAlert('Failed to check if BCeID already exists in provider organization: ' + userName, error)
+        }
+      }
+    },
+
+    /**
+     * Check if user already exists in provider organization by first name, last name, and email.
+     */
+    async doesUserExist(firstName, lastName, email) {
+      try {
+        const res = await ApiService.apiAxios.get(`${ApiRoutes.ORGANIZATIONS_USERS.replace(':organizationId', this.userInfo.organizationId)}?firstName=${firstName}&lastName=${lastName}&email=${email}`);
+        if (Array.isArray(res.data) && res.data.length >= 1) {
+          return true
+        }
+        return false
+      } catch (error) {
+        if (error.response.status === 404) {
```

Comment:
If we make the backend update to remove 404 this won't be necessary.

Reply:
Removed.